skins/oasis/modules/ContentDisplayController.class.php

```diff
@@ -1,23 +1,23 @@
 <?php
+
 /**
  * Helper Functions: adds picture attribution info and replaces section edit links with pencil icon and link
  * Fixme: this is not really a proper module
  * @author Maciej Brencz
  */
-
 class ContentDisplayController extends WikiaController {
 	/**
 	 * Show section edit link for anons (RT #79897)
 	 */
-	static function onShowEditLink(&$parser, &$showEditLink) {
+	static function onShowEditLink( &$parser, &$showEditLink ) {
 		global $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)